packages/web/app/src/pages/target-laboratory.tsx

```diff
@@ -242,6 +243,40 @@ function Save(props: {
   );
 }
 
+const onModifyHeaders: ComponentProps<typeof GraphiQL>['onModifyHeaders'] = async (
```

Comment:
I will create PR for graphiql, this is very useful for folks, maybe a better name? this is to modify headers before execution